Os/Posix/File.cpp

```diff
@@ -0,0 +1,279 @@
+// ======================================================================
+// \title Os/Posix/File.cpp
+// \brief posix implementation for Os::File
+// ======================================================================
+#include <cerrno>
+#include <fcntl.h>
+#include <unistd.h>
+#include <limits>
+
+#include <Os/File.hpp>
+#include <Os/Posix/File.hpp>
+#include <Fw/Types/Assert.hpp>
+#include <Os/Posix/errno.hpp>
+
+namespace Os {
+namespace Posix {
+namespace File {
+
+// Os specific direct writing support
+#ifdef __linux__
+#define DIRECT_FLAGS O_DIRECT
```

Comment:
What is this for? It doesn't look like DIRECT_FLAGS is used anywhere.

Reply:
Direct flags were used when we supported the direct sync mode, which we have removed.